Backend/router/v1/report.ctrl.js

```diff
@@ -1,120 +1,122 @@
 const models = require('../../models');
 
 const createReport = (req, res) => {
-  if (req.isAuthenticated()) {
-    models.report.create({
-      what: req.body.data[0].what,
-      location: req.body.data[0].location,
-      time: req.body.data[0].time,
-      who: req.body.data[0].who,
-      details: req.body.data[0].details,
-      userID: req.user[0].dataValues.id
-    }).then((result) => {
-      res.status(201);
-      res.json({
-        msg: "Report is made successfully"
-      });
-    });
-  } else {
-    res.status(403);
-    res.json({
-      msg : "You are not logged in"
-    });
+  if (!req.isAuthenticated()) {
+    res.status(403).json({msg: "You are not logged in"});
+    return ;
   }
+  models.report.create({
+    what: req.body.data[0].what,
+    location: req.body.data[0].location,
+    time: req.body.data[0].time,
+    who: req.body.data[0].who,
+    details: req.body.data[0].details,
+    userID: req.user[0].dataValues.id
+  }).then((result) => {
+    res.status(201);
+    res.json({id: result.dataValues.id});
+  }).catch((err) => {
+    res.json({msg: "Error: failed to create report"});
+  });
 };
 
 const showReportList = (req, res) => {
-  if (req.isAuthenticated()) {
-    models.report.findAll({
-      where: {
-        userID: req.user[0].dataValues.id
-      }
-    }).then((result) => {
-      res.status(200);
-      res.json(result);
-    }).catch((err) => {});
-  } else {
-    res.status(401)
-    res.json({
-      msg: "get report failed"
-    })
+  if (!req.isAuthenticated()) {
+    res.status(403).json({msg: "You are not logged in"});
+    return ;
   }
+  models.report.findAll({
+    where: {
+      userID: req.user[0].dataValues.id
+    }
+  }).then((result) => {
+    res.status(200);
+    res.json(result);
+  }).catch((err) => {
+    res.json({msg: "Error: failed to load all reports"});
```

Comment:
Same here, is there a status code for this error case?

Reply:
Same below.

Reply:
There is no http error code. The application error code will be handled by another issue soon. ( #60 )